docs/release/trg-0/trg-4-02.md

```diff
@@ -0,0 +1,133 @@
+---
+title: TRG 4.02 - Base Images
+---
+
+| Status | Created     | Post-History                                                                    |
+|--------|-------------|---------------------------------------------------------------------------------|
+| Draft  | 15-Jan-2024 | Add Base Image Security                                                         |
+| Active | 06-Dec-2023 | Add advantages of Dockerhub images                                              |
+| Draft  | 04-May-2023 | Remove notice in favor of dedicated TRG; Add mandatory base image for frontends |
+| Active | 25-Nov-2022 | Initial release                                                                 |
+
+## Why
+
+As part of our legal due diligence, we need to provide the best information possible about our distributed (published)
+Docker images. Similar to our 3rd-party dependency scans and the `DEPENDENCIES` file, Docker images also have to be
+scanned and the results published. We want to help you to keep a high standard process, by defining guidelines,
+described in this TRG.
+
+## Description
+
+As Eclipse Tractus-X project, we don't have automated processes for publishing container scan results (yet). This is why
```

Comment:
would remove the (yet)

Reply:
sure no problem, but can we change it now? feels a bit strange for me to write about upcomming maybe plans for the future into a TRG

Reply:
I won't change this, as it is not the subject of this PR to change other content of the TRG. Even though the change is as little as this one.